osi_object.proto

```diff
@@ -54,6 +54,15 @@ message Vehicle
     /// The id of the driver of the (ego) vehicle.
     /// Field need not be set if ego_vehicle is set to false.
     optional Identifier driver = 10;
+    
+    /// The vector pointing from the bounding box center point (base.position) to the middle (in x, y and z) 
+    /// of the rear axle under neutral load conditions.
+    optional Vector3d bbcenter_to_rear = 11;
+
+	/// The vector pointing from the bounding box center point (base.position) to the middle (in x, y and z) 
+	/// of the front axle under neutral load conditions.    		
+	optional Vector3d bbcenter_to_front = 12;
+    
 
```

Comment:
> optional Vector3d relative_ego_origin = 6;

@SoykaDaimler and @AndreasVIRES 
The description above states, that this is the vector to the rear axle, if the car is a simulated car. This is now duplicated? We could consider removing it and replacing it by the two new definitions?